### PR TITLE
Fix uncaught exceptions in examples.

### PR DESCRIPTION
**Pull request by zhangkungmail**

### CODE REVIEW

File: examples/src/main/java/org/apache/pdfbox/examples/lucene/LucenePDFDocument.java [hunk 1]
```diff
@@ -360,7 +361,10 @@ private void addContent(Document document, InputStream is, String documentLocati
             {
                 stripper = new PDFTextStripper();
             }
-            stripper.writeText(pdfDocument, writer);
+            try {
+                stripper.writeText(pdfDocument, writer);
+            } catch (final PdfTimeoutException e) {
```
Comment (ashutoshvsingh) [inline examples/src/main/java/org/apache/pdfbox/examples/lucene/LucenePDFDocument.java:366]: why is exception being swallowed

Comment (zhangkungmail) [reply examples/src/main/java/org/apache/pdfbox/examples/lucene/LucenePDFDocument.java:366]: We already discussed. Exception is only thrown on the production issue call path.

Kun Zhang [2:51 PM] 
Ash, do you want to populate the exception to all callers of the bad function? Or only populate it on the specific path where the problem happened

Ashutosh Singh [3:01 PM] 
you can start with the problem path

**Comment by ashutoshvsingh** [comment]
It is just not a good practice to swallow exception just propagate it up

**Comment by zhangkungmail** [comment]
Let's discuss. The exception is thrown in a deep function call. If we populate exceptions at all places, 100 files need to be changed.

**Comment by ashutoshvsingh** [comment]
👍

**Pull request merged**